user/homework.go

```diff
@@ -46,7 +48,7 @@ func NewHomework(problems []*api.Problem, c *config.Config) *Homework {
 		hw.Items = append(hw.Items, item)
 	}
 
-	hw.template = fmt.Sprintf("%%%ds %%s\n", hw.maxTitleWidth())
+	hw.template = fmt.Sprintf("%%s%%s %%s\n")
```

Comment:
As there are no substitutions, you could now reduce this to:

```go
hw.template = "%s%s %s\n"
```

The previous versions generation of the template padding using `%20s` or whatever value besides 20 the actual max length feels more correct. That being said it definitely needed to be rewritten to be more understandable.